### PR TITLE
:D

### PR DESCRIPTION
**Pull request by NoobSaiyan**

### CODE REVIEW

File: src/components/detailed-info-card/detailed-info-card.styles.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const DetailedInfoCardContainer = styled.div`
   height: 100%;
   width: 150px;
   justify-self: center;
-  margin-right: 5%;
+  margin-right: 10%;
```
Comment (NoobSaiyan) [inline src/components/detailed-info-card/detailed-info-card.styles.jsx]: dont accept this part

Comment (Trugamr) [inline src/components/detailed-info-card/detailed-info-card.styles.jsx]: ?

Comment (NoobSaiyan) [reply src/components/detailed-info-card/detailed-info-card.styles.jsx]: sorry sir i ll make that correct asap

**Comment by NoobSaiyan** [comment]
;-;

**Pull request closed without merging**